packages/cloud-cognitive/src/components/SidePanel/SidePanel.js

```diff
@@ -555,10 +576,10 @@ SidePanel = pkg.checkComponentEnabled(SidePanel, componentName);
 
 SidePanel.validatePageContentSelector =
```

Comment:
Since we've updated the prop to be able to use either `selectorPageContent` _or_ `pageContentSelector`, should this be updated as well to include both?

Reply:
I did think about this, but since `pageContentSelector` is being deprecated I think it's best to only validate `selectorPageContent` because we don't want people using `pageContentSelector` anymore. Adding the `deprecateProps` helper function displays a warning message in the console, if `pageContentSelector` is being used, saying this prop is being deprecated and to use `selectorPageContent` instead.

![image](https://user-images.githubusercontent.com/10215203/120645958-0628ee80-c447-11eb-9f42-0ef2ba6dfe8f.png)